examples/quickstarts/helidon-quickstart-se/src/main/java/io/helidon/examples/quickstart/se/Main.java

```diff
@@ -52,8 +53,10 @@ public static void main(String[] args) {
      * Updates HTTP Routing and registers observe providers.
      */
     static void routing(HttpRouting.Builder routing) {
+        SeOpenApiFeature openApi = SeOpenApiFeature.builder().build();
         GreetService greetService = new GreetService();
-        routing.register("/greet", greetService)
+        routing.addFeature(openApi)
```

Comment:
I did not inline `ObserverFeature.create()`; it was that way from others' earlier changes. 

I try for minimal changes to the existing code and try to minimize other, gratuitous changes that might disrupt consistency with other code (other examples in this case) and possibly muddle the reason for the change. If we want to clean up stylistic choices across multiple examples let's do that but in its own PR rather than hit-or-miss in unrelated PRs.